skipped_tests.toml

```diff
@@ -111,16 +125,19 @@ unit = [
 	"tests/unit/utils/test_templates.py::RenderTestCase::test_render_mako_variable",
 ]
 integration = [
+	"tests/integration/loader/test_ext_modules.py",  # Make pytest to stuck in sle15sp1 classic pkg
 	"tests/integration/modules/test_cmdmod.py::CMDModuleTest::test_which_bin",
+	"tests/integration/ssh/test_state.py::SSHStateTest::test_state_running",  # This test makes no sense with new locking mechanism for Salt SSH
 	"tests/pytests/integration/master/test_clear_funcs.py::test_clearfuncs_config",
-	"tests/pytests/integration/master/test_clear_funcs.py::test_fileroots_write",
 	"tests/pytests/integration/master/test_clear_funcs.py::test_fileroots_read",
+	"tests/pytests/integration/master/test_clear_funcs.py::test_fileroots_write",
 	"tests/pytests/integration/netapi/rest_tornado/test_minions_api_handler.py::test_get_no_mid",
 	"tests/pytests/integration/netapi/rest_tornado/test_root_handler.py::test_simple_local_post",
 	"tests/pytests/integration/netapi/rest_tornado/test_root_handler.py::test_simple_local_post_only_dictionary_request",
 	"tests/pytests/integration/runners/state/orchestrate/test_events.py::test_jid_in_ret_event",
 	"tests/pytests/integration/runners/state/orchestrate/test_events.py::test_orchestration_with_pillar_dot_items",
 	"tests/pytests/integration/runners/state/orchestrate/test_events.py::test_parallel_orchestrations",
+	"tests/pytests/integration/ssh/test_master.py::test_service",  # Failing when running on containers. dbus not available
```

Comment:
Skipped in code in https://github.com/openSUSE/salt/pull/616/commits/3ada27711a7431b84d3e82f1f848aa684e9f075c